Makefile

```diff
@@ -269,3 +269,7 @@ geth-windows-amd64:
 	$(GORUN) build/ci.go xgo -- --go=$(GO) --targets=windows/amd64 -v ./cmd/geth
 	@echo "Windows amd64 cross compilation done:"
 	@ls -ld $(GOBIN)/geth-windows-* | grep amd64
+
+
+e2e_istanbul_coverage:
```

Comment:
I don't see this being run in CI, I think you need to add it to the list of jobs